src/bench/Cargo.toml

```diff
@@ -50,28 +33,16 @@ tokio = { version = "0.2", package = "madsim-tokio", features = [
     "time",
     "signal",
 ] }
-tokio-stream = { workspace = true }
-toml = "0.8"
-tracing = "0.1"
-tracing-subscriber = "0.3.17"
 
 [target.'cfg(not(madsim))'.dependencies]
 workspace-hack = { path = "../workspace-hack" }
 
 [target.'cfg(target_os = "linux")'.dependencies]
 nix = { version = "0.29", features = ["fs", "mman"] }
 
-[[bin]]
-name = "s3-bench"
-path = "s3_bench/main.rs"
-
 [[bin]]
 name = "sink-bench"
 path = "sink_bench/main.rs"
```

Comment:
cc @xxhZs is `sink_bench` still useful?

Reply:
Occasionally, I use it to test the performance of a particular sink